### ISSUE TITLE
Unification of SOAP|REST request|response handling via ExamplesTable

### ISSUE THREAD

**Issue by AmyKosinova** [EmbedITCZ/jbehave-support#91]
While creating collection we use different notation **collection.order.attribute** VS **collection[order].attribute**. We should unify this.

```
Given [NameRequest] data for [TEST]:
| name                           | data  |
| name                           | test  |
| addressList.addressInfo.0.city | Praha |
```

```
When [POST] request to [TEST]/[user/] is sent with data:
| name                 | data           | contextAlias      |
| firstName            | Pedro          | FIRST_NAME        |
| lastName             | Salgado        | LAST_NAME         |
| addresses[0].country | Brazil         | ADDRESS_0_COUNTRY |
```

**Comment by paveljandejsek** [comment]
This is related only to requests? Or also to responses?

**Comment by AmyKosinova** [comment]
> This is related only to requests? Or also to responses?

Good question. Response verifying / parsing should be uniform as well.

**Comment by paveljandejsek** [comment]
> Response verifying / parsing should be uniform as well.

And is it already? Or did you not check so we dont know?

**Comment by AmyKosinova** [comment]
Responses are handled own way, so this needs to be addressed as well.

**Comment by paveljandejsek** [comment]
Solved by #158 and #159
